ripple

```diff
@@ -93,7 +93,7 @@ case ${1} in
     ;;
     "run")
         cargo build --features local_dev
-        DEVICE_HOST=${2} cargo run --features local_dev core/main 
+        DEVICE_HOST=${2} cargo run --features local_dev --bin ripple
```

Comment:
Unrelated: `main` branch now has some of the multiple-binary/services stuff so we need to update this script.